### ISSUE TITLE
Command Line option for auth-file

### ISSUE THREAD

**Issue by rbeede** [duplicati/duplicati#2312]
I have:
- [x ] searched open and closed issues for duplicates

----------------------------------------

### Version info
**Duplicati Version:** master (and 2.0.1.45_experimental_2017-02-03)
**Operating System:** Windows

### Bug description
It would be nice to add another option to  duplicati.commandline.exe backup  to complement the  --auth-password and --auth-username options.

Passing an argument with a password on the command line exposes the password in clear-text on the system to other users (process viewer).

Add an additional argument:

--auth-file=/path/to/file.creds

The file should be two lines:
username
password

The user should be informed in help.txt to ensure they put proper file ACLs to protect the password.

Encryption of the file content or password could be a separate feature request but can be out-of-scope for this one.

**Comment by kenkendk** [comment]
You can do this with `--parameterfile=/path/to/file.txt`.
In the file you can set your options, like:
```
--auth-username=username
--auth-password=password
--dblock-size=100MB
```

Does this fix your issue?

**Comment by rbeede** [comment]
Yes, could an example be added to the documentation mentioning this method?

Something like this under the auth-password documentation:

If you don't want the password to show up in the command line on the OS process list you can use the --parameterfile=/path/to/file.txt option.   This also allows for secure ACLs to be set to protect the password.
